bucket/robo3t.json

```diff
@@ -1,13 +1,13 @@
 {
-    "version": "1.4.2",
+    "version": "1.4.4",
     "description": "The free lightweight GUI for MongoDB enthusiasts",
     "homepage": "https://robomongo.org",
     "license": "GPL-3.0-only",
     "architecture": {
         "64bit": {
-            "url": "https://download.studio3t.com/robomongo/windows/robo3t-1.4.2-windows-x86_64-8650949.zip",
-            "hash": "6f4123c7a1d5ee7121d5f1af942e48140bb6e40008040d4caa17878324e34050",
-            "extract_dir": "robo3t-1.4.2-windows-x86_64-8650949"
+            "url": "https://download.studio3t.com/robomongo/windows/robo3t-1.4.4-windows-x86_64-e6ac9ec5.zip",
```

Comment:
This would be the GitHub download link

Usually we prefer GitHub

https://github.com/Studio3T/robomongo/releases/download/v1.4.4/robo3t-1.4.4-windows-x86_64-e6ac9ec5.zip

Reply:
@pratikpc Done.